lib/console/view.coffee

```diff
@@ -53,6 +55,10 @@ class ConsoleView extends ScrollView
     else
       @addBeforeInput item, opts
 
+  limitHistory: (maxSize = 1000) ->
+    if @items.childNodes.length > maxSize
```

Comment:
Maybe make this a `while` instead of `if`, so that if something happens that causes multiple messages to get inserted for a single call to `limitHistory` it will still work to keep the list limited. I don't think that can happen now, but it's the sort of thing that could easily happen due to error or future refactoring.